concent_api/core/templates/admin/subtasks_with_downloads.html

```diff
@@ -0,0 +1,4 @@
+<div class="line_decorator"></div>
+<div class="subtasks_with_downloads"><span>
```

Comment:
I am far behind what is going on currently in frontend world, but shouldn't the <span> be in next line, with its content end finishing tag?

Reply:
Done.